config/logger.ts

```diff
@@ -0,0 +1,7 @@
+import * as winston from "winston";
+
+export const logger = winston.createLogger({
+    level: 'info',
+    format: winston.format.json(),
+    transports: [new winston.transports.File({filename: 'log'})]
```

Comment:
extract all data you can to config file (json).
your logs should not be in project folder! but somewhere in c://romach-v/log